File: app/src/main/java/com/cabe/app/novel/activity/AboutActivity.java
package com.cabe.app.novel.activity;

import android.os.Bundle;

import com.cabe.app.novel.R;

public class AboutActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        setTitle("关于我们");
    }
}

File: app/src/main/java/com/cabe/app/novel/activity/AboutActivity.kt
package com.cabe.app.novel.activity

import android.os.Bundle
import com.cabe.app.novel.R

class AboutActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        title = "关于我们"
    }
}